source/com.android.billingclient/billing/Additions/Additions.cs

```diff
@@ -121,7 +116,9 @@ public Task<QueryPurchaseHistoryResult> QueryPurchaseHistoryAsync(QueryPurchaseH
                 })
             };
 
-            QueryPurchaseHistory(queryPurchaseHistoryParams, listener);
+            // QueryPurchaseHistory method may not exist in v8.x, commenting out for now
+            // QueryPurchaseHistory(queryPurchaseHistoryParams, listener);
+            tcs.TrySetResult(new QueryPurchaseHistoryResult { Result = null, PurchaseHistoryRecords = null });
```

Comment:
Review what API is missing here, is the C# binding missing it? Or do we need to call a new API.

Docs are at: https://developer.android.com/reference/com/android/billingclient/api/package-summary

If you can't figure out the appropriate change needed here, comment what your hypothesis is.